_build_cfg.yml

```diff
@@ -191,8 +191,8 @@ Topics:
     File: application_health
   - Name: Events
     File: events
-  - Name: Output Build Dependencies
-    File: output_deps
+  - Name: Dependency Tree Generation
+    File: dependecy_tree_generation
```

Comment:
@adellape I would lean toward Building Dependency Trees based on that intro paragraph:

"Build a dependency tree for the specified image stream......"

However, I'd be okay with either of these over what we have currently.